src/lib/util/calcMaxTripLength.ts

```diff
@@ -34,6 +34,9 @@ export function calcMaxTripLength(user: MUser, activity?: activity_type_enum) {
 		case 'Raids':
 		case 'TheatreOfBlood':
 		case 'TombsOfAmascut':
+		case 'CamdozaalMining':
+		case 'CamdozaalSmithing':
+		case 'CamdozaalFishing':
```

Comment:
Don't think it makes sense for HP to give these a boost imo?

Reply:
fixed